packages/wonder-blocks-tokens/src/tokens/media-queries.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * A default set of media queries to use for different screen sizes.
+ *
+ * Breakpoint documentation: https://khanacademy.atlassian.net/wiki/spaces/WB/pages/2099970518/Layout+Breakpoints
+ *
+ */
+
+export const pureWidths = {
+    xsMax: 567,
+    smMin: 568,
+    smMax: 681,
+    mdMin: 682,
+    mdMax: 1023,
+    lgMin: 1024,
+} as const;
+
+export const breakpoints = {
```

Comment:
**suggestion:** We tend to use singular for defining tokens, so in this case it would be better something like `breakpoint`, `mediaQuery` or `mq`.

You can see some existing examples here:
https://github.com/Khan/wonder-blocks/blob/main/packages/wonder-blocks-tokens/src/tokens/border.ts
https://github.com/Khan/wonder-blocks/blob/main/packages/wonder-blocks-tokens/src/tokens/typography.ts

Reply:
Done!